marshmallow_jsonapi/schema.py

```diff
@@ -76,7 +76,22 @@ class Meta:
         pass
 
     def __init__(self, *args, **kwargs):
+        self.include_data = kwargs.pop('include_data', ())
         super(Schema, self).__init__(*args, **kwargs)
+        for field_name, field in self.fields.items():
+            if field_name in self.include_data:
+                if not isinstance(field, BaseRelationship):
+                    raise ValueError('Can only include relationships. "%s" is a "%s"'
```

Comment:
Minor: for consistency with the rest of the codebase, use `str.format` rather than `%` for string formatting.